pyproject.toml

```diff
@@ -81,10 +81,9 @@ docs = [
 ]
 lint = [
     "flake8>=3.5.0",
-    "flake8-comprehensions",
-    "flake8-bugbear",
     "flake8-simplify",
     "isort",
+    "ruff",
```

Comment:
we should pin this, as ruff is moving very fast and PRs will break because of changes not related to the PRs (eg. #11086)